### PR TITLE
feat: day1 training session uploaded

### PR DESCRIPTION
**Pull request by Rajharsh2**
- Day1 session upoaded
- assigment uploaded please do it.

### CODE REVIEW

File: session1.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+Day 1: Git fundamentals (03-Aug-2023)
```
Comment (manvithalva12) [inline session1.md:1]: raj can please add some comment lines and explain the procedures so that it will be easy to catch up.

**Review by manvithalva12 — CHANGES_REQUESTED** [review]

File: session1.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+Traditional way of maintaining work in the form of folders and archives is not a scalable solution and is extremely error prone. 
+Programmers identified this problem and came up with version control systems - which are special softwares written to help with the problem of versioning of work.
+Git was invented by Linus Torvalds to help maintain the source code of Linux systems.
+All systems these days come with a default installation of git. You can check if git is installed by running: 
+
+git --version
+git version 2.32.0 (Apple Git-132)
```
Comment (albin1616) [inline session1.md:9]: change apple to windows

**Review by albin1616 — CHANGES_REQUESTED** [review]

File: session1.md (hunk 1 above)
Comment (Rajharsh2) [reply session1.md:1]: resolved

File: session1.md (hunk 2 above)
Comment (Rajharsh2) [reply session1.md:11]: resolved

**Review by manvithalva12 — APPROVED** [review]

**Review by albin1616 — APPROVED** [review]

**Pull request merged**